libs/native-federation/src/builders/build/builder.ts

```diff
@@ -303,10 +312,16 @@ function transformIndexHtml(
 }
 
 function addDebugInformation(fileName: string, rawBody: string): string {
-  if (fileName !== '/remoteEntry.json') {
-    return rawBody;
+  if (fileName === '/remoteEntry.json') {
+    return addRemoteEntryDebugInformation(rawBody);
   }
+  if (fileName === '/importmap.json') {
```

Comment:
Also update the generated `importmap.json` with the debug information like we do for the `remoteEntry.json`